deploy/sample-backup-configs/sample_config_build.ini

```diff
@@ -12,56 +12,37 @@ postbuildhook=
 [builds]
 # this section references builds defined in config_build_recipes.ini
 # if you add a build here, it will be built when you run buildafi
-# Legacy recipe without an L2
 firesim-quadcore-nic-llc4mb-ddr3
-#firesim-singlecore-no-nic-l2-lbp
-#firesim-singlecore-nic-l2-lbp
-#firesim-quadcore-no-nic-l2-lbp
-#firesim-quadcore-nic-l2-lbp
-firesim-quadcore-no-nic-l2-llc4mb-ddr3
-firesim-quadcore-nic-l2-llc4mb-ddr3
-#fireboom-singlecore-no-nic-l2-lbp
-fireboom-singlecore-no-nic-l2-llc4mb-ddr3
-#fireboom-singlecore-nic-l2-lbp
-fireboom-singlecore-nic-l2-llc4mb-ddr3
-#firesim-supernode-singlecore-nic-llc4mb-ddr3
-#firesim-supernode-quadcore-nic-llc4mb-ddr3
+firesim-quadcore-no-nic-llc4mb-ddr3
+fireboom-singlecore-no-nic-llc4mb-ddr3
```

Comment:
So wait, is the l2 removed from this? Or did you just shorten the name?

Reply:
Shortened names

Reply:
I would prefer to keep l2 in the names. What do you think @davidbiancolin

Reply:
This also saves us from having to change all those inis later in this PR, right? And people are going to copy paste these anyway, so there's basically no savings

Reply:
We've decided to keep the l2 in the naing